soc/arm/st_stm32/stm32l0/dts_fixup.h

```diff
@@ -149,4 +149,10 @@
 #define DT_ADC_1_CLOCK_BITS		DT_ST_STM32_ADC_40012400_CLOCK_BITS_0
 #define DT_ADC_1_CLOCK_BUS		DT_ST_STM32_ADC_40012400_CLOCK_BUS_0
 
+#define DT_DAC_1_BASE_ADDRESS		DT_ST_STM32_DAC_40007400_BASE_ADDRESS
+#define DT_DAC_1_NAME			DT_ST_STM32_DAC_40007400_LABEL
```

Comment:
Outside of DAC name, we don't need the other defines, please remove.

Reply:
Yes, the `DT_INST_` defines are also generated in devicetree.conf, e.g.:
```
DT_ST_STM32_DAC_40007400_LABEL="DAC_1"
DT_INST_0_ST_STM32_DAC_LABEL="DAC_1"
```
But I just implemented it similar to other drivers like the ADC, which don't rely on `DT_INST_` defines.

So you suggest to completely remove dts_fixups and let the user (and the sample) use the long `DT_INST_0_ST_STM32_DAC_LABEL` instead of `DT_DAC_1_NAME`?

Reply:
I'm not very fond of this way of doing, but the only way to go with DT_INST macros are to create a board alias and reference it in sample.
You can check `tests/drivers/i2c` for reference.

Reply:
Add an alias in the board's `.dts` file and use the macro generated by that alias in the **sample** (`DT_ALIAS_DAC_`)  but use `DT_INST_` macros in the **driver** itself.